src/faq.njk

```diff
@@ -178,7 +167,7 @@ description: Up-to-date list of things people of asked us about NZ Tech Rally 20
 
     <div class="card accent-border">
       <p>
-        We only accept debit or credit card payments on Stripe. We do not have the capacity to manage manual bank transfers right now.
+        More info coming soon, stay tuned.
```

Comment:
Was this changed because we're keen to allow manual bank transfer payment for NZ Tech Rally 2024?